src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -19,6 +19,7 @@
 #include "distributed/citus_safe_lib.h"
 #include "distributed/connection_management.h"
 #include "distributed/listutils.h"
+#include "distributed/lock_graph.h"
```

Comment:
do we really need this ?

Reply:
For this **_ParseIntField()_**

Reply:
Then seems that it exports something that it's not supposed to do. Anyway, this is not related to this pr.